src/managers/sessionManager/page/SessionManager.ts

```diff
@@ -193,7 +193,12 @@ export class SessionManager implements ISessionManager {
       this.context.environmentInfo.isBrowserAndSupportsServiceWorkers ||
       this.context.environmentInfo.isUsingSubscriptionWorkaround
     ) {
-      this.setupSessionEventListeners();
+      if (this.context.environmentInfo.canTalkToServiceWorker) {
```

Comment:
This seems backwards, if we can talk to the sw shouldn't we call `this.setupSessionEventListeners()` to setup the message with the sw?

Reply:
I think you're right. Fixed